terrarun/server/__init__.py

```diff
@@ -1977,11 +1969,10 @@ def check_permissions_get(self, current_user, current_job, run_id=None,):
     def _get(self, current_user, current_job, run_id=None):
         """Return run information"""
         if not run_id:
-            return {}, 404
+            return {"title": "Parameter run_id must be provided."}, 404
```

Comment:
Did this format come from anywhere in particular?
If it's something you've decided, could you add a comment as such?
Every response format is currently based on API docs for Terraform Cloud

Reply:
Haha, no. It's based on my mistaked interpretation of the docs. I should have nested it way more:
```{
  "errors": [
    {
      "status": "404",
      "title": "Parameter run_id must be provided."
    }
  ]
}```

Or used an ApiError like in other places. 
```{
  "errors": [ApiError(title = "Parameter run_id must be provided.", detail = "Parameter run_id must be provided.", status = 404)]
}```

But mostly it's not related to the print stuff so I shouldn't have committed it :)